src/themes/pug.css

```diff
@@ -13,7 +13,10 @@ body[data-theme="pug"] {
 
   .product-detail-image {
     border-radius: 100%;
-    max-width: 300px;
+    
+    @media screen and (min-width: 576px) {
```

Comment:
This causes the same as mentioned above. I think we should remove this so the image could be full-width.

Reply:
@WiNloSt see https://github.com/metabase/shoppy/pull/68#discussion_r1790181614, the idea is that the image is only full width below 576px, and cap the width after that.

Reply:
Fixed in https://github.com/metabase/shoppy/pull/68/commits/c34eb0cec20ce88c3c4fde2cbe0d09f62499ef95